packages/gasket-core/lib/gasket.js

```diff
@@ -38,7 +38,7 @@ export class Gasket {
     // prune nullish and/or empty plugins
     config.plugins = config.plugins
       .filter(Boolean)
-      .map(plugin => plugin.default || plugin) // quality of life for cjs apps
+      .map(plugin => 'default' in plugin ? plugin.default : plugin) // quality of life for cjs apps
```

Comment:
TypeScript is able to use an `in` check to narrow a type; seeing if a `default` property is falsy doesn't do the same, so the old expression wasn't able to be resolved as a `Plugin` object.